resources/rules/post_identifier_merge/entity/type/entity_type_propogation.clj

```diff
@@ -0,0 +1,8 @@
+;; -------------------------------------------
```

Comment:
We are deprecating denotesSubClassOf, so this rule will likely get deleted entirely.